app/routes/goldlab.js

```diff
@@ -10,6 +10,18 @@ export default Ember.Route.extend({
     });
   },
   actions: {
+    addShade(palette, color, shade_value) {
+      var shade = this.store.createRecord('shade', {
+        value: shade_value,
+        weight: 0,
```

Comment:
This should be the default value on the model and not set here if the weight is unknown at this point.